src/python/pose_format/bin/directory.py

```diff
@@ -79,6 +111,21 @@ def get_corresponding_pose_path(video_path: Path, keep_video_suffixes: bool = Fa
     return video_path.with_suffix(".pose")
 
 
+def process_video(vid_path: Path, keep_video_suffixes: bool, pose_format: str, additional_config: dict) -> bool:
+    print(f'Estimating {vid_path} on CPU {psutil.Process().cpu_num()}')
+    try:
+        pose_path = get_corresponding_pose_path(video_path=vid_path, keep_video_suffixes=keep_video_suffixes)
+        if pose_path.is_file():
+            print(f"Skipping {vid_path}, corresponding .pose file already created.")
```

Comment:
Here's an example where logging.debug might be good, as this could potentially output thousands of messages